static/charts.js

```diff
@@ -0,0 +1,88 @@
+/* globals $, Chart */
+var curChart;
+
+function setChart(labels, openedData, closedData, type) {
+    var ctx = document.getElementById("canvas");
+
+    curChart = new Chart(ctx, {
+        type: 'line',
+        data: {
+            labels: labels,
+            datasets: [{
+                label: "Issues Opened",
+                backgroundColor: "RGBA(33, 150, 243, 0.2)",
+                borderColor: "RGBA(33, 150, 243, 1)",
+                data: openedData,
+                fill: true,
+            }, {
+                label: "Issues Closed",
+                backgroundColor: "RGBA(244, 67, 54, 0.2)",
+                borderColor: "RGBA(244, 67, 54, 1)",
+                data: closedData,
+                fill: true,
+            }]
+        },
+        options: {
+            responsive: true,
+            title: {
+                display: true,
+                text: 'Community Activity'
+            },
+            tooltips: {
+                mode: 'index',
+                intersect: false,
+            },
+            hover: {
+                mode: 'nearest',
+                intersect: true
+            },
+            scales: {
+                xAxes: [{
+                    display: true,
+                    scaleLabel: {
+                        display: true,
+                        labelString: type
+                    }
+                }],
+                yAxes: [{
+                    display: true,
+                    scaleLabel: {
+                        display: true,
+                        labelString: 'Number'
+                    }
+                }]
+            }
+        }
+    });
+}
+
+function updateChart(type) {
+    if(curChart){ curChart.destroy(); }
+
+    $.getJSON("/static/activity-data.json",
+        function(data) {
+            var labels = data.year.labels;
+            var openedData = data.year.opened;
```

Comment:
L65 and 66 are not needed right, its getting overwritten later inside if else anyway

Reply:
Fixed.